app/models/amusement_park.rb

```diff
@@ -1,3 +1,8 @@
 class AmusementPark < ApplicationRecord
   has_many :rides
+  has_many :mechanics
```

Comment:
An amusement park does not have a direct connection to a mechanic and the only way to get there is `through` `:rides`.

```ruby
has_many :rides
has_many :mechanics, through: :rides
```

This is one reason why I recommend creating a dbdiagram.io schema so you can actually see the table connections which might make the associations easier to see.